server/base/src/main/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -254,7 +255,7 @@ public int getMaxConcurrentMigrations() {
   }
 
   @Override
-  public void init(ServerConfiguration conf) {
+  public void init(ServerConfigurationFactory conf) {
```

Comment:
The [JIRA ticket](https://issues.apache.org/jira/browse/ACCUMULO-4535) has more info, but there are two `init` functions in the base class.  The framework that initializes the load balancer calls the `ServerConfigurationFactory` function instead of this one.